code/builders/builder-manager/src/utils/template.ts

```diff
@@ -54,5 +54,6 @@ export const renderHTML = async (
       PREVIEW_URL: JSON.stringify(previewUrl, null, 2), // global preview URL
     },
     head: (await customHead) || '',
+    ignorePreview,
```

Comment:
Don't you need to update the types and the CLI definitions etc?

Reply:
Which types? and what CLI definitions? This option (`ignorePreview`) already gets set this just passes it to the manager template.